orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/ExpressionTransform.java

```diff
@@ -266,7 +272,18 @@ static String escapeSimpleExpression(String expression) {
    */
   private String includeExecutionParameter(String e) {
     String expression = e;
-    for (String fn : EXECUTION_AWARE_FUNCTIONS) {
+
+    // An expression aware function is any that takes an Execution as its first parameter
+    Set<String> expressionAwareFunctions = new HashSet<>(EXECUTION_AWARE_FUNCTIONS);
```

Comment:
Wouldn't mind splitting the other stage-related functions out of `ExpressionSupport` and out of the hardcoded `EXECUTION_AWARE_FUNCTIONS` constant ... was thinking that would be a subsequent PR (I don't _think_ anything should break!).